beast/physicsmodel/grid.py

```diff
@@ -186,22 +152,62 @@ def __getattr__(self, name):
             raise AttributeError(msg.format(type(self).__name__, name))
 
     def __getitem__(self, name):
-        if hasattr(self.grid, "read"):
-            try:
-                return self.grid.read(field=name)
-            except TypeError:
-                return self.grid[name]
-        else:
-            return self.grid[name]
+        return self.grid[name]
 
     def copy(self):
         """ returns a copy of the object """
         return self.__class__(backend=self._backend.copy())
 
 
+class SEDGrid(ModelGrid):
+    """
+    Generate a grid that the full observational model (SEDs).
+    Currently a directy interface to ModelGrid.  Setup for later expansion.
+
+    Attributes
+    ----------
+    seds : ndarray
+        2D `float` array (# models, # bands) giving the seds
+
+    lamb : ndarray
+        1D `float` array of the wavelengths of the sed bands
+
+    filters : list
+        list of the filter names of the sed bands
+
+    grid : astropy.Table
```

Comment:
satrapy.table

Reply:
I did figure out how to reference the astropy.table.Table class with a link!  So now clicking takes one to the Table class API.  Yeah.